scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -73,10 +74,35 @@
 groups_to_aggregate = args.aggregateGroups
 
 if args.unfolding:
+
     unfolding_axes, unfolding_cols = differential.get_pt_eta_axes(args.genBins[0], template_minpt, template_maxpt, args.genBins[1])
     datasets = unfolding_tools.add_out_of_acceptance(datasets, group = "Wmunu")
     groups_to_aggregate.append("BkgWmunu")
 
+elif args.addHelicityHistos:
+
```

Comment:
Maybe the code below can go into differential.get_theoryAgnostic_axes or so?

Reply:
Maybe, but I am not sure I want to put it into differential, which was born to include things for the unfolding. Presumably the theory agnostic should stay a bit more independent in its own scripts for the moment

Reply:
Ok, yes can maybe go into theoryAgnostic_tools?

Reply:
I am doing a mixture eventually, the axis are defined in differential, but for most other function there will be a new theoryAgnostic_tools.py, although I can probably further simplify.